### PR TITLE
Add a feature to re-run a submission by challenge host

### PR DESCRIPTION
**Pull request by Sanji515**
#### Changes proposed in this pull request:

- Added a new option in the settings menu option for re-running the submission

- Added new endpoint for re-running the submission

- Link to live demo: http://pr-175-evalai.surge.sh

### CODE REVIEW

**Comment by Sanji515** [comment]
@Shekharrajak @lunayach @galipremsagar @RishabhJain2018 

Please review

File: src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts [hunk 1]
```diff
@@ -414,4 +414,34 @@ export class ChallengeviewallsubmissionsComponent implements OnInit, AfterViewIn
...
+   * Modal to confirm the submission re-run
+   * @param submissionId submission id
+   */
+  reRunSubmission(submissionId) {
+    const SELF = this;
+
+    SELF.apiCall = () => {
+      const BODY = {};
+      SELF.apiService.postUrl(
+        SELF.endpointsService.reRunSubmissionURL(submissionId), BODY
```
Comment (lunayach) [inline src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts]:
```suggestion
        SELF.endpointsService.reRunSubmissionURL(submissionId), BODY)
```
(Close it here)

Comment (Sanji515) [reply src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts]: done

**Review by lunayach — APPROVED** [review]
LGTM! Was able to do a re-run for a submission.

**Pull request merged**